src/App.js

```diff
@@ -1,14 +1,24 @@
 import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
```

Comment:
Hey @sherribooher can you leave App.css in there

Reply:
Should be good for PR now. Sorry about that.